src/main/features/core/applicationMenu.js

```diff
@@ -125,7 +125,10 @@ const template = [
       },
       {
         label: 'Close',
-        accelerator: 'CmdOrCtrl+W',
+        accelerator: (() => {
```

Comment:
I apologize, following existing coding style is always a good thing, looks like I made some terrible code back in the day 😆